docs2/site/docs/getting-started/schema-types.md

```diff
@@ -120,14 +120,44 @@ The name of each member _is_ the value.
 
 GraphQL.NET provides two methods of defining GraphQL enums.
 
-You can use `EnumerationGraphType<TEnum>` to automatically generate values by providing a .NET
-`enum` for `TEnum`. The `Name` will default to the .NET Type name, which you can override in
-the constructor. The `Description` will default to any `System.ComponentModel.DescriptionAttribute`
-applied to the enum type. The `DeprecationReason` will default to any `System.ObsoleteAttribute`
-applied to the enum type. By default, the name of each enum member will be converted to CONSTANT_CASE.
-Override `ChangeEnumCase` to change this behavior. Apply a `DescriptionAttribute` to an enum member
-to set the GraphQL `Description`. Apply an `ObsoleteAttribute` to an enum member to set the GraphQL
-`DeprecationReason`.
+I. You can use `EnumerationGraphType<TEnum>` to automatically generate values by providing a .NET
```

Comment:
I would not use "I" and "II", especially as the primary/outer numbering.  I would use 1/2/3 first and a/b/c second.  Note that when I and II and etc are used, it is always done lowercase.  "i." and "ii." and "iii.".

Reply:
Default Microsoft Word numbering pattern:
![image](https://user-images.githubusercontent.com/6377684/147127071-03e39f02-46d1-433c-a700-a483dd33e4aa.png)

Reply:
reformatted